### ISSUE TITLE
JSApplicationIllegalArgumentException: Error while updating property 'lineHeight' in shadow node of type: RCTAztecView

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#12211]
**Sentry Url**: https://sentry.io/share/issue/0244926c0adf4aeda025bc36bae98d3e/
**User Count**: 936
**Count**: 4073
**First Release**: org.wordpress.android-14.9-rc-1
**First Seen**: 2020-05-21T22:24:51Z
**Last Seen**: 2020-06-16T22:57:57Z
**24 Hours**: 224
**30 Days**: 4030

```
JSApplicationIllegalArgumentException: Error while updating property 'lineHeight' in shadow node of type: RCTAztecView
     at com.facebook.react.uimanager.ViewManagersPropertyCache$FloatPropSetter.getValueOrDefault(ViewManagersPropertyCache.java:223)
     at com.facebook.react.uimanager.ViewManagersPropertyCache$PropSetter.updateShadowNodeProp(ViewManagersPropertyCache.java:106)
     at com.facebook.react.uimanager.ViewManagerPropertyUpdater$FallbackShadowNodeSetter.setProperty(ViewManagerPropertyUpdater.java:159)
     at com.facebook.react.uimanager.ViewManagerPropertyUpdater.updateProps(ViewManagerPropertyUpdater.java:63)
     at com.facebook.react.uimanager.ReactShadowNodeImpl.updateProperties(ReactShadowNodeImpl.java:318)
     at com.facebook.react.uimanager.UIImplementation.createView(UIImplementation.java:249)
     at com.facebook.react.uimanager.UIManagerModule.createView(UIManagerModule.java:465)
     at java.lang.reflect.Method.invoke(Method.java:null)
     at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:371)
     at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:150)
...
(7 additional frame(s) were not displayed)
```

**Comment by bummytime** [comment]
@maxme or @hypest any thoughts here? (ref: `RCTAztecView `)

**Comment by hypest** [comment]
Took me a bit but managed to replicate the issue. Happens when the a block has the `line-height`  css property set.

Example html:
```
<!-- wp:heading {"align":"left","level":4,"className":"has-primary-background-color has-background","style":{"typography":{"lineHeight":"2.5"}}} -->
<h4 class="has-text-align-left has-primary-background-color has-background" style="line-height:2.5">Heading with line-height set</h4>
<!-- /wp:heading -->
```

@marecar3 , can you please prioritize working on this? line-height is not currently supported by the Aztec Wrapper (RCTAztecView) and we need to somehow guard or add support for it so the app doesn't crash when the block has that prop from the web.

**Comment by marecar3** [comment]
> Took me a bit but managed to replicate the issue. Happens when the a block has the `line-height` css property set.
> 
> Example html:
> 
> ```
> 
> <h4 class="has-text-align-left has-primary-background-color has-background" style="line-height:2.5">Heading with line-height set</h4>
> 
> ```
> 
> @marecar3 , can you please prioritize working on this? line-height is not currently supported by the Aztec Wrapper (RCTAztecView) and we need to somehow guard or add support for it so the app doesn't crash when the block has that prop from the web.

Will do, thanks!

**Comment by hypest** [comment]
Fix will be included in editor release v1.31.0, due to land in v15.2 of WPAndroid.
